### PR TITLE
Bug 1387107 - Stop and start spark session between experiment runs

### PR DESCRIPTION
**Pull request by fbertsch**
The job was failing intermittently. We've seen this before on
consecutive spark jobs, for example, main_summary.

### CODE REVIEW

**Comment by fbertsch** [comment]
@sunahsuh currently running on a cluster, I'm hopeful it will finish about the same time the build does.

File: src/main/scala/com/mozilla/telemetry/views/ExperimentAnalysisView.scala [hunk 1]
```diff
@@ -24,33 +24,41 @@ object ExperimentAnalysisView {
...
     spark.sparkContext.setLogLevel("INFO")
+    spark
+  }
+
+  def main(args: Array[String]) {
+    val conf = new Conf(args)
+    var spark = getSpark
 
-    val data = spark.read.parquet(conf.inputLocation())
+    var data = spark.read.parquet(conf.inputLocation())
```
Comment (sunahsuh) [inline src/main/scala/com/mozilla/telemetry/views/ExperimentAnalysisView.scala]: Looks like if we rename just this initial instance of `spark` and `data` (and the call to `getExperiments()` then we can avoid mutable vars

**Comment by fbertsch** [comment]
@sunahsuh RFAL

**Comment by sunahsuh** [comment]
Fingers crossed!

**Comment by fbertsch** [comment]
@sunahsuh ran on a cluster, took <1 hour, which seems to be what we'd expect. Let's kick this off!

**Pull request merged**